packages/group/src/index.ts

```diff
@@ -144,10 +144,10 @@ export class Group {
      * @param nodes The stringified JSON of the group.
      * @returns The {@link Group} instance.
      */
-    static import(exportedGroup: string): Group {
+    static import(nodes: string): Group {
```

Comment:
to decide whether to use `nodes` or `exportedGroup`, I chose to reuse what is in `@zk-kit/imt`

https://github.com/privacy-scaling-explorations/zk-kit/blob/8f37c779dae20f13a7a31cc7bed0728b6a07d52c/packages/imt/src/lean-imt/lean-imt.ts#L336